srcpkgs/webkit2gtk/template

```diff
@@ -132,7 +132,30 @@ pre_configure() {
 	esac
 }
 
+post_configure() {
+	mkdir -p webkit2gtk-40 webkit2gtk-50
+	(
+		cd webkit2gtk-40 && \
+		configure_args="${configure_args} -DUSE_SOUP2=ON -DENABLE_WEBDRIVER=OFF" \
+		do_configure
+	)
+
+	(
+		cd webkit2gtk-50 && \
```

Comment:
```suggestion
		cd webkit2gtk-50
```
or is there a reason why we should use `&&` @ahesford?

Reply:
Not really, I'm just in the habit of using `&&` in these constructs to fail fast and avoid side effects.

Reply:
any error in the template should stop the build. If the subshell inherits that property, I think && could be removed.